docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -15,32 +15,31 @@ This quickstart guides you through deploying a local instance of Airbyte Self-Ma
 
 This quickstart shows you how to:
 
-- [Install abctl](#part-1-install-abctl)
-- [Run Airbyte](#part-2-run-airbyte)
-- [Set up authentication](#part-3-set-up-authentication)
+- [Install Docker Desktop](#part-1-install-docker-desktop)
```

Comment:
I am not sure if this is already required but pretty soon, docker hub is going to require an account in order to pull images. I wonder if we want to add something explicitly about it.